.github/workflows/ci.yml

```diff
@@ -62,7 +62,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v4
-      - uses: pnpm/action-setup@v3
+      - run: corepack enable
```

Comment:
Could you explain why this was necessary as this would be different from all other jobs? Maybe Windows? Should we change this in some other places?

Reply:
I changed it on some of the iterations to fix things on Windows. I'll try removing it to see if it actually is needed for Windows to work of if I just forgot to undo it before.